### PR TITLE
Update OneLogin

### PR DESCRIPTION
**Pull request by fletcherdodd**
This also contributes to #6845

### CODE REVIEW

File: entries/o/onelogin.com.json [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+    "custom-software": [
+      "OneLogin Protect",
+      "Duo",
+      "RSA SecurID",
+      "Symantec VIP"
+    ],
+    "custom-hardware": [
+      "Yubico OTP"
+    ],
+    "documentation": "https://support.onelogin.com/kb/4266569",
```
Comment (hkamran80) [inline entries/o/onelogin.com.json:20]:
```suggestion
    "documentation": "https://support.onelogin.com/kb/4266569/add-multi-factor-authentication",
```

The URL without the last bit kept throwing errors for me.

File: entries/o/onelogin.com.json [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
       "sms",
       "totp",
       "custom-hardware",
       "u2f"
     ],
-    "documentation": "https://onelogin.service-now.com/support?id=kb_article&sys_id=43d99143db109700d5505eea4b96197a",
+    "custom-software": [
+      "OneLogin Protect",
+      "Duo",
+      "RSA SecurID",
```
Comment (hkamran80) [inline entries/o/onelogin.com.json:14]: I can't see any mention of SecurID on their help page

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/o/onelogin.com.json (hunk 2 above)
Comment (fletcherdodd) [reply entries/o/onelogin.com.json:14]: In step 3 of enabling authentication factors, there's a list below the screenshot. If you click on 'RADIUS' under partners, it's listed there. RADIUS adds support for it

**Review by hkamran80 — APPROVED** [review]

**Comment by hkamran80** [comment]
Thank you! Your contribution will be visible in around 10 minutes, although it may take longer due to server and browser caches.

**Comment by fletcherdodd** [comment]
Thank you, you'll need to mark off OneLogin on the list in the mentioned issue, I don't think I have the access.

**Pull request merged**